src/components/Book.js

```diff
@@ -1,19 +1,45 @@
+// Book.js
+
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { addBook, removeBook } from '../redux/Books/booksSlice';
+import { checkStatus } from '../redux/categories/categoriesSlice';
+
+function Book({
+  id, title, author,
+}) {
+  const dispatch = useDispatch();
+
+  const handleAddBook = () => {
+    dispatch(addBook({
+      id, title, author,
+    }));
+  };
+
+  const handleRemoveBook = () => {
+    dispatch(removeBook(id));
+  };
```

Comment:
- [ ] Great job for adding the method but I think it is not working as expected when I try to press on remove nothing is removed since it should be removed as required so kindly resolve this issue as well here 🔥💯🤩🤩

###  Please remeber the add button and check status here meaningless since the add book will be below beside the form and check status in the category page but here there is another buttons 

![image](https://user-images.githubusercontent.com/58553711/223876237-eab5b82e-37bf-4962-b79d-c9ff23fcb6a5.png)

### These buttons you can add it without its functionality fow now 🤩

![image](https://user-images.githubusercontent.com/58553711/223876561-5f1d1a1a-5657-469a-a09f-25a3074cb29d.png)